ENIGMAsystem/SHELL/Graphics_Systems/General/GSscreen.cpp

```diff
@@ -154,7 +154,7 @@ static inline int draw_tiles()
 void clear_view(float x, float y, float w, float h, float angle, bool showcolor)
 {
   if (enigma::d3dMode && enigma::d3dPerspective)
-    d3d_set_projection_perspective(x, y, w, h, angle);
+    d3d_set_projection_perspective(x, y, w, h, 40);
```

Comment:
Yes, I meant to leave it, that's the default field of view angle that GM8 uses for `d3d_set_perspective` so it must stay. If the user wants a custom fov, they should set the projection manually in the draw event.

The GM8 manual states the following:
http://gamemaker.info/en/manual#d3d_set_projection_ortho
>The additional parameters work as follows. If you specified the camera position, point to look at, and up vector, you can still change how wide the lens of the camera is. This is called the field of view. A reasonable value is somewhere between 40 and 45 degrees.

Reply:
Nope, I've now fixed it: c86f0604cd4664d95b7572e7a893b9e417cd86ef

This was a mistake I made in #1388 with respect to the default field of view. I don't know how I didn't see it in the GM8 manual, but the angle parameter of both `d3d_set_projection_ortho` and `d3d_set_projection_perspective` is an angle used to rotate the entire screen. So for example you could rotate the picture upside down this way. This is why this pull request is good, because we are eliminating this inconsistencies and ensuring we're doing the right thing.

http://gamemaker.info/en/manual#d3d_set_projection_ortho